zwave-classifier.js

```diff
@@ -23,6 +23,22 @@ const COMMAND_CLASS_METER = 50;               // 0x32
 //const COMMAND_CLASS_SWITCH_ALL = 39;        // 0x27
 const COMMAND_CLASS_CONFIGURATION = 112;    // 0x98
 
+const QUIRKS = [
+  {
+    identifiers: {
+      manufacturerId: '0x0086',
+    },
+    excludeProperties: ['current'],
```

Comment:
I may need to tweak this, but I think its fine for now. I'll test the ZW099 module and see if current works for it.